### ISSUE TITLE
[Splatoon] High resolution shadows makes models look blocky

### ISSUE THREAD

**Issue by slashiee** [slashiee/cemu_graphic_packs#149]
![image](https://user-images.githubusercontent.com/15317421/34089675-dd662e5a-e366-11e7-84e5-197000f6af13.png)
![image](https://user-images.githubusercontent.com/15317421/34089683-eebf3c50-e366-11e7-9c94-2e4f1861b188.png)

**Comment by Crementif** [comment]
Might be a problem with reflections, which are really low res. It needs some mip scaling (256x256), but I'm not quite sure if I know how to do that without breaking stuff.

**Comment by Crementif** [comment]
Can't find a way to replicate the bug, it looks fine wherever I checked. Does this only happen on the match end screen?

**Comment by slashiee** [comment]
It also appears on the end screen for any single player map.
![](https://i.redd.it/tea8sb9cxqlz.png)

**Comment by Crementif** [comment]
The texture that might be causing an issue is a 4x4 texture, so you might want to upscale (or remove) that. Editing the shaders didn't do anything for me for some reason.

**Comment by Crementif** [comment]
Fixed the issue with commit https://github.com/slashiee/cemu_graphic_packs/commit/91bdffd0dae455479c738c5a8e8568d31c4450ce supposedly.

**Comment by slashiee** [comment]
Might be better to put it into main pack like what we agreed with in #143

**Comment by Crementif** [comment]
I don't think it has something to do with the blurring at the edges, it's probably just not scaling properly. If I had to guess it's still a 4x4 texture that needs scaling, but that's way to common to attempt. I don't know why he only tried scaling this shader, it's only used on the end-screen? I had to replay the story mission to dump this shader which only was there when I reached the end-screen.

Anyway, I'll put it in there, there's probably no harm in scaling those shaders.
